layouts/partials/labs.html

```diff
@@ -0,0 +1,80 @@
+<section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 1 - Introduction to Falco</h2>
+    <p>General introduction to Falco in single Linux hosts. Learn how to install Falco with Docker, and detect threads
```

Comment:
General introduction to Falco running in single Linux hosts. Learn how to install Falco with Docker, and detect threads, such as: